### PR TITLE
Feature/FOUR-11539: Process Launchpad - Tabs - Default

### PR DESCRIPTION
**Pull request by CarliPinell**
## Issue & Reproduction Steps

As a Designer, I want to access to all the past requests, with the detailed status so it easier for me to access.
This is the default view without information to show
st requests, with the detailed status so it easier for me to access.

image

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-11539

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-cc9c51dc9a.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-cc9c51dc9a.eng.processmaker.net

File: resources/js/processes-catalogue/components/ProcessTab.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template>
+    <div  class="mt-3">
+        <b-tabs content-class="mt-3 text-style">
+            <b-tab title="My Requests" active>
+                <div class="d-flex align-items-center justify-content-center flex-column">
```
[the remark this answers is not in the document]
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/ProcessTab.vue]: solved

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-cc9c51dc9a.eng.processmaker.net

File: resources/img/launchpad-images/frame.png
Comment (fagubla) [inline resources/img/launchpad-images/frame.png:1]: this must be an SVG, then the name must describe

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/DefaultTab.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    imageSrc: {
+      type: String,
+      default: "/img/launchpad-images/frame.png",
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/DefaultTab.vue:21]: please rehuse the default: "/img/tasks-loading.svg",

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/img/launchpad-images/frame.png
Comment (pmPaulis) [reply resources/img/launchpad-images/frame.png:1]: This new files is not necesary

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/DefaultTab.vue (hunk 2 above)
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/DefaultTab.vue:21]: solved

File: resources/img/launchpad-images/frame.png
Comment (CarliPinell) [reply resources/img/launchpad-images/frame.png:1]: solved changing to /img/tasks-loading.svg

**Review by fagubla — APPROVED** [review]

**Pull request merged**